habitat_baselines/config/default.py

```diff
@@ -69,6 +69,19 @@
 _C.RL.SUCCESS_REWARD = 2.5
 _C.RL.SLACK_REWARD = -0.01
 # -----------------------------------------------------------------------------
+# preemption CONFIG
+# -----------------------------------------------------------------------------
+_C.RL.preemption = CN()
+# Append the slurm job ID to the resume state filename if running a slurm job
+# This is useful when you want to have things from a different job but same
+# same checkpoint dir not resume.
+_C.RL.preemption.append_slurm_job_id = False
```

Comment:
@mathfac @Skylion007 what are your thoughts on the default values here?  Currently this is set to do resuming always, but I can also make the default values the old functionality.